data/data-pipeline/data_pipeline/etl/score/etl_utils.py

```diff
@@ -108,3 +110,174 @@ def floor_series(series: pd.Series, number_of_decimals: int) -> pd.Series:
     )
 
     return floored_series
+
+
+def _create_df_from_yaml_contents(
+    fields_list_from_yaml: list,
+    fields_to_store_in_codebook: list,
+) -> pd.DataFrame:
+    """Helper function to create a dataframe from yaml fields
+
+    This function does:
+        1. Creates a dictionary that will eventually become a dataframe.
```

Comment:
when is "eventually"?

Reply:
at the end of the functioin. updated comment.